code/modules/mining/materials.dm

```diff
@@ -91,20 +91,41 @@ var/global/list/initial_materials	//Stores all the matids = 0 in helping New
 		return
 	addAmount(mat_id,-amount)
 
+/datum/materials/proc/removeSheetAmount(mat_id, amount)
```

Comment:
Consider using "number" instead of "amount" - in the rest of the materials code, amount is used for CCs.